artview/components/navigator.py

```diff
@@ -47,7 +465,7 @@ def __init__(self, Vradar=None, Vgrid=None, Vfilelist=None,
             Parent instance to associate to this class.
             If None, then Qt owns, otherwise associated w/ parent PyQt instance
         '''
-        super(FileNavigator, self).__init__(name=name, parent=parent)
+        super(FileNavigator_old, self).__init__(name=name, parent=parent)
         self.central_widget = QtWidgets.QWidget()
```

Comment:
just a copy I made before start changing everything. This and other things should be removed before merge